Makefile

```diff
@@ -73,11 +77,8 @@ docker-push: docker-build
 	docker push $(IMAGE_NAME):$(IMAGE_VERSION)
 
 clean:
-	rm -rf .tmp
-	rm -rf phase3/.tmp
-	rm -rf phase1/gce/.tmp
-	rm -rf phase1/azure/.tmp
-	rm -rf phase1/vsphere/.tmp
+	rm -rf phase3/*/.tmp/
+	rm -rf phase1/*/*/
```

Comment:
I understand, it is scary. cluster_name comes from config file. `make clean` may be called even when the config file does not exist and `make clean` may behave unexpectedly.
I am also thinking of writing the cluster_name to some hidden file when configured and make clean can take the cluster_name from the file and clean. In scenarios where we deploy multiple clusters this may look complex.